### PR TITLE
added NYU Abu Dhabi cs faculty and updated affiliations

### PR DESCRIPTION
**Pull request by bishnudev01**
(Please check #6814, I am getting _institution not in institutions.csv_ error but this file should be generated automatically I suppose)

1. Added NYU Abu Dhabi to the list of institutions and the CS faculty.
2. Updated the following outdated affiliations (These faculty members are now at NYU Abu Dhabi):
_Azzam Mourad_
Lebanese American University -> NYU Abu Dhabi
_Keith W. Ross_
NYU Shanghai -> NYU Abu Dhabi
_Talal Rahwan_
Masdar Institute -> NYU Abu Dhabi

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

### CODE REVIEW

File: csrankings-m.csv [hunk 1]
```diff
@@ -157,6 +157,7 @@ Mahshid Mojtabavi Naeini,Texas Tech University,https://www.depts.ttu.edu/provost
 Mai Anh Tien,Singapore Management University,https://sis.smu.edu.sg/faculty/profile/167371/MAI-Anh-Tien,5TgvaiwAAAAJ
 Mai ElSherief,Univ. of California - San Diego,https://mayelsherif.github.io/melsherif,Rykl2HkAAAAJ
 Mai Fadel,King Abdulaziz University,http://mfadel.kau.edu.sa/Default.aspx?Site_ID=0011432&Lng=EN,NOSCHOLARPAGE
+Mai Oudah,NYU Abu Dhabi,https://nyuad.nyu.edu/en/academics/divisions/science/faculty/mai-oudah.html,PUDnxbMAAAAJ
```
Comment (emeryberger) [inline csrankings-m.csv]: Lecturer, not TT faculty; please remove.

File: csrankings-a.csv [hunk 2]
```diff
@@ -2437,8 +2437,9 @@ Azer Bestavros,Boston University,http://www.bu.edu/cs/azer-bestavros,S5B0QmMAAAA
 Aziz Mohaisen,University of Central Florida,http://cs.ucf.edu/~mohaisen,q387AywAAAAJ
 Aziz Salah,Université du Québec à Montréal,https://www.labunix.uqam.ca/~salah_a/3ardh.php,NOSCHOLARPAGE
 Aznul Qalid Md Sabri,University of Malaya,https://umexpert.um.edu.my/aznulqalid,oX5-84kAAAAJ
+Azza Abouzied,NYU Abu Dhabi,https://nyuad.nyu.edu/en/academics/divisions/science/faculty/azza-abouzied.html,RkddXygAAAAJ
 Azzam Ibrahim,University of Jordan,http://eacademic.ju.edu.jo/azzam.sleit,yZ42G9YAAAAJ
-Azzam Mourad,Lebanese American University,http://sas.lau.edu.lb/csm/people/azzam-mourad.php,p3DTnQQAAAAJ
+Azzam Mourad,NYU Abu Dhabi,https://nyuad.nyu.edu/en/academics/divisions/science/faculty/azzam-mourad.html,p3DTnQQAAAAJ
```
Comment (emeryberger) [inline csrankings-a.csv]: Not found here: https://nyuad.nyu.edu/en/academics/faculty.html?_charset_=UTF-8&division=science&program=computer-science-major&affiliation=NYU+Abu+Dhabi&keywords=

Comment (emeryberger) [reply csrankings-a.csv]: (Home page is dead)

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-m.csv (hunk 1 above)
Comment (bishnudev01) [reply csrankings-m.csv]: removed

File: csrankings-a.csv (hunk 2 above)
Comment (bishnudev01) [reply csrankings-a.csv]: fixed

**Comment by bishnudev01** [comment]
The changes are not reflected on the website (csrankings.org). Is there any reason why?

**Pull request merged**